src/libs/Browser/index.web.js

```diff
@@ -72,7 +72,8 @@ function isSafari() {
  */
 function openRouteInDesktopApp(shortLivedAuthToken = '', email = '') {
     const params = new URLSearchParams();
-    params.set('exitTo', `${window.location.pathname}${window.location.search}${window.location.hash}`);
+    const openingFromDesktopRedirect = window.location.pathname === '/desktop-signin-redirect';
```

Comment:
Let's use `ROUTES.DESKTOP_SIGN_IN_REDIRECT` here?

Reply:
@amyevans I added a comment to explain the check and prepended a `/`, which exists on the pathname!